WangscapeTest/TestRasterImage.cpp

```diff
@@ -0,0 +1,105 @@
+#include <gtest/gtest.h>
+
+#include <noise/RasterImage.h>
+#include <noise/module/WangscapeModules.h>
+
+class TestRasterImage : public ::testing::Test
+{
+protected:
+    noise::module::GradientX gradientXModule;
+    noise::module::GradientY gradientYModule;
+    noise::module::Const constModule;
+    sf::Image image;
+    TestRasterImage()
+    {
+        constModule.SetConstValue(0.);
+    };
+    ~TestRasterImage() {};
```

Comment:
Looks like it probably came from copying `TestCornerCombiner`. I'll check if there are any other test classes that need a destructor removed or `default`ed.